.gitignore

```diff
@@ -175,3 +175,73 @@ modules.xml
 /installer/deps
 
 # End of https://www.gitignore.io/api/phoenix
+
```

Comment:
This is getting a little out of hand. It's unreasonable to think that every kind of file might be captured in the `.gitignore`. Best to have a more disciplined git workflow and review process.